ios/src/TGMapViewController.h

```diff
@@ -49,6 +49,7 @@ NS_ASSUME_NONNULL_BEGIN
 @optional
 - (void)mapView:(TGMapViewController*)mapView didLoadSceneAsync:(NSString*)scene;
 - (void)mapView:(TGMapViewController*)mapView didSelectFeature:(NSDictionary*)feature atScreenPosition:(CGPoint)position;
+- (void)didCompleteView:(TGMapViewController *)mapView;
```

Comment:
Nitpick: just to be consistent with the rest of Cocoa, this should probably read more like:
`- (void)viewDidCompleteLoading(TGMapViewController*)mapView`